src/main/java/com/powsybl/network/conversion/server/NetworkConversionException.java

```diff
@@ -1,11 +1,36 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
 package com.powsybl.network.conversion.server;
 
+import java.util.Objects;
+
 /**
  * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
  */
-class NetworkConversionException extends RuntimeException {
+public final class NetworkConversionException extends RuntimeException {
+
+    public enum Type {
+        UNSUPPORTED_FORMAT
+    }
+
+    private final Type type;
 
-    NetworkConversionException(String message, Throwable cause) {
-        super(message, cause);
+    private NetworkConversionException(Type type, String msg) {
```

Comment:
same no need for type if there's one type

Reply:
same reason